tests/test_analysis.py

```diff
@@ -70,6 +70,15 @@ def test_pyoptsparse(self):
         os.chdir("pyoptsparse")
         subprocess.check_call(["python", "rosenbrock.py"])
 
+    def test_wing_opt_SLSQP(self):
+        # first copy files
+        os.chdir("aero")
+        shutil.copy("../ffd/ffd.xyz", ".")
+        shutil.copy("../../aero/analysis/wing_vol_coarsen.cgns", "wing_vol_coarsen.cgns")
+        shutil.rmtree("output", ignore_errors=True)
+        cmd = ["python", "aero_opt.py"]
+        subprocess.check_call(mpiCmd + cmd + gridFlag + SLSQP)
```

Comment:
There are a lot of duplicated lines in these tests. Would it make sense to create a parameterized class?